substrate/client/rpc-api/src/statement/mod.rs

```diff
@@ -27,6 +27,8 @@ pub mod error;
 #[rpc(client, server)]
 pub trait StatementApi {
 	/// Return all statements, SCALE-encoded.
+	///
+	/// Note! The max RPC response size is 15 MiB, so this call may fail if there are too many.
```

Comment:
All these notes do not really make sense to me. Why are you here talking about some response limit that is configured somewhere else? And below you are raising questions?